docs/user/index.md

```diff
@@ -0,0 +1,137 @@
+# User & Programmers Manual
+
+This document describes how to use Perseo CEP.
+
+## Content
+
+-   [Introduction](#introduction)
+-   [Perseo Rules](#perseo-rules)
+-   [Actions](#actions)
+-   [String substitution syntax](#string-substitution-syntax)
+-   [Metadata and object values](#metadata-and-object-values)
+-   [Examples](#examples-of-epl-rules)
+
+## Introduction
+
+Perseo is an [Esper-based](http://www.espertech.com/esper/) Complex Event Processing (CEP) software designed to be fully
+_NGSIv2_-compliant. It uses NGSIv2 as the communication protocol for events, and thus, Perseo is able to seamless and
+jointly work with _context brokers_ such as [Orion Context Broker](https://github.com/telefonicaid/fiware-orion).
```

Comment:
Same here...

Reply:
Fixed in https://github.com/telefonicaid/perseo-fe/pull/592/commits/ce3e52c048d89a72045670ba717cf35c190fa4e3